### PR TITLE
A couple of small WebGL fixes

### PR DESCRIPTION
**Pull request by nox**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/21142)

### CODE REVIEW

File: components/script/dom/webglrenderingcontext.rs [hunk 1]
```diff
@@ -3863,27 +3863,14 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
...
-        // From the GLES 2.0.25 spec:
-        //
-        //     "If either width or height is greater than the value of
-        //      MAX_RENDERBUFFER_SIZE , the error INVALID_VALUE is
-        //      generated."
-        //
-        // and we have to throw out negative-size values as well just
-        // like for TexImage.
-        //
-        // FIXME: Handle max_renderbuffer_size, which doesn't seem to
```
Comment (emilio) [inline components/script/dom/webglrenderingcontext.rs]: The quote is nice I think, but if you don't think it's worth it it's fine for me too.

**Review by emilio — APPROVED** [review]

File: components/script/dom/webglrenderingcontext.rs (hunk 1 above)
Comment (nox) [reply components/script/dom/webglrenderingcontext.rs]: The spec links are more important than the quotes IMO.

**Comment by nox** [comment]
@bors-servo r=emilio

I just realised this makes even more tests pass.

**Pull request merged**